services/cognitiveservices/v2.0/luis/authoring/models.go

```diff
@@ -1978,7 +1978,9 @@ func (vi VersionInfo) MarshalJSON() ([]byte, error) {
 	if vi.AssignedEndpointKey != nil {
 		objectMap["assignedEndpointKey"] = vi.AssignedEndpointKey
 	}
-	objectMap["externalApiKeys"] = vi.ExternalAPIKeys
+	if vi.ExternalAPIKeys != nil {
+		objectMap["externalApiKeys"] = vi.ExternalAPIKeys
```

Comment:
I saw that these fields indeed have `omitEmpty` in the tags but it bytes us in the custom marshaling, oh well. I guess we dont have a test case in the test server for this ? :(